src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -19,6 +19,22 @@ import config from 'config';
 import restApiSlice from 'src/shared/state/api-slices/restSlice';
 
 import type { BlastSettingsConfig } from 'src/content/app/tools/blast/types/blastSettings';
+import type { Species } from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
+import type { BlastSubmission } from '../blast-results/blastResultsSlice';
+
+export type BlastSubmissionPayload = {
+  species: Species[];
+  querySequences: string[];
+  parameters: Record<string, string>;
+};
+
+type BlastSubmissionResponse = {
+  submissionId: string;
+  jobs: Array<{
```

Comment:
Should this match with the one here?
https://github.com/Ensembl/ensembl-client/pull/719/files#diff-efd6c7c3fa16b35020ea90f17187298bc502e22972a024c3e5347e43be2d32c2R54